static/js/util/api.js

```diff
@@ -1,4 +1,4 @@
-// @flow
+// @flow weak
```

Comment:
Not sure exactly what you mean, I agree that we might want to have `string|null` instead of just `string` for some of these fields, but then it's a question of how much work we put in to getting the typechecker to pass. I changed `end_date` on work history to be `string|null` and that seems to be passing.